functional-tests/bdd-tests/CRS_request_relocation.feature

```diff
@@ -0,0 +1,15 @@
+﻿Feature: CRS_request_relocation
+    As a logged in business user
+    I would like to request a Cannabis Retail Store relocation
+
+Scenario: Request CRS Relocation
+    Given the CRS application has been approved
+    And I am logged in to the dashboard as a private corporation
+    And I click on the Licences tab
+    And the licence fee has been paid
+    And I click on the Request Relocation link
+    And I review the account profile
+    And I submit a licence relocation application
+    And I complete the payment
+    And I click on the Licences tab
+    Then **outcome to be confirmed**
```

Comment:
The outcome here will be tricky to test here. The outcome is the creation of a CRS Relocation app that's available on the dashboard. Final approval of the application will be an update to the address displayed on the licence; I think it is enough to validate that the application can be submitted for now, until we've got Dynamics testing sorted out.